src/CanvasWrapper.js

```diff
@@ -1,11 +1,29 @@
 //@ts-check
-const MathHelper = require('./MathHelper')
-const Vector2 = require('./Vector2')
-const Line = require('./Line')
-const Vertex = require('./Vertex')
-const Ring = require('./Ring')
-const ThemeManager = require('./ThemeManager')
-const { getChargeText } = require('./UtilityFunctions')
+import Line         from './Line'
+import MathHelper   from './MathHelper'
+import Ring         from './Ring'
+import ThemeManager from './ThemeManager'
+import Vector2      from './Vector2'
+import Vertex       from './Vertex'
+
+/**
+ * Translate the integer indicating the charge to the appropriate text.
+ * @param {Number} charge The integer indicating the charge.
+ * @returns {String} A string representing a charge.
+ */
+function getChargeText(charge) {

```

Comment:
Consolidated and bugfixed from a `CanvasWrapper` member function and `UtilityFunctions.js`.